spec/controllers/polls_controller_spec.rb

```diff
@@ -184,33 +180,44 @@
   describe 'GET #edit' do
     let(:poll) { FactoryGirl.create(:valid_poll) }
 
-    it 'should be succes' do
+    it 'should be success' do
       expect(get :edit, id: poll.id).to be_succes
```

Comment:
I think this is alias there, because it's work both ways succes/success, but successs, for example, fails as expected.